app/javascript/packs/find-out.js

```diff
@@ -64,7 +64,7 @@ const onDropdownChange = () => {
       topic_area: dropdownSelections[1].innerText,
     };
     fetchTaggings(dropdownsObject);
-    $('div#find-out__overlay').removeClass('find-out__overlay');
+    $('#find-out__overlay').removeClass('find-out__overlay');
```

Comment:
`find-out__overlay` is treated as both as class and an ID; would it be possible to change one of these for clarity without plunging down a rabbithole?